randovania/games/dread/logic_database/Cataris.txt

```diff
@@ -1669,7 +1669,9 @@ Extra - asset_id: collision_camera_024
       Lay Small Bomb or Shoot Beam
   > Dock to Kraid Arena
       All of the following:
-          Before Kraid and Can Slide
+          Any of the following:
+              Before Kraid and Can Slide
+              Morph Ball and After Kraid and After Artaria - ArbEnky Blob East and After Cataris - Diffusion Tutorial Blob 2 and After Cataris - Blob above Kraid and After Cataris - Kraid Eyedoor Ceiling Blob and After Cataris - Kraid Eyedoor Blob
```

Comment:
Artaria's probably isn't needed here I'm sure, but why are the Above Kraid and Eyedoor ceiling blobs needed if they don't have any influence on the backdoor path?

Reply:
My thought process is that I want it to take this path: 
![image](https://github.com/user-attachments/assets/7c683664-446a-427f-880a-368ee1fb406a)
Using this path instead of jumping over the lava pool seemed reasonable, because there seems to be many different options to do that part. Comparatively, requiring the extra blobs seems like a reasonable shorthand here. It's a coincidennce that this path doesn't go through any doors. The Kraid Eyedoor is excluded from randomization, so I don't think there would be anything wrong with doing that.